scripts/apps/vocabularies/views/vocabulary-config.html

```diff
@@ -34,24 +34,38 @@
     </div>
 
     <div class="sd-page__content">
-      <div ng-if="loading" class="loading-overlay" ng-class="{active: loading}" style="opacity: 0.5;"></div>
-        <ul class="pills-list">
-          <li ng-repeat="vocabulary in vocabularies | orderBy:'display_name|translate' | filter: query track by vocabulary._id" ng-if="matchFieldTypeToTab(tab, vocabulary.field_type)">
-            <h6>{{ vocabulary.display_name | translate }}</h6>&nbsp;
-            <em><strong ng-show="!vocabulary.field_type">({{ vocabulary.items.length }})</strong></em>
-            <div class="actions">
-              <button
-                ng-click="openVocabulary(vocabulary)"
-                title="{{:: 'Edit vocabulary' | translate }}"
-              ><i class="icon-pencil"></i></button>
-              <button
-                ng-click="remove(vocabulary)"
-                ng-if="tab !== 'vocabularies'"
-                title="{{:: 'Remove vocabulary' | translate }}"
-              ><i class="icon-trash"></i></button>
+        <div ng-if="loading" class="loading-overlay" ng-class="{active: loading}" style="opacity: 0.5;"></div>
+
+        <div ng-repeat="tag in tags" ng-if="existsVocabulariesForTag(tag, tab)">
```

Comment:
Don't bother changing it for this one, but I think we should always prefer the first. Not only it's more effective since you only iterate the group once instead of one time per item, but it's cleaner to read and understand. At least to me.

```
for group in groups
    {{group.label}}

    for item in group
        {{item.label}}
```

```
for item in items
    for group in groups
        if item in group
            {{item.label}}
```